connectors/src/lib/data_sources.ts

```diff
@@ -343,3 +347,10 @@ export function renderGfmMarkdownSection(
 
   return top;
 }
+
+export function renderGfmMarkdownSection(
```

Comment:
Can we kill that bad boy and call the other one directly from the github code :pray:

Reply:
Fixed.